Tests/SourceKitLSPTests/WorkspaceTestDiscoveryTests.swift

```diff
@@ -679,23 +655,16 @@ final class WorkspaceTestDiscoveryTests: XCTestCase {
       tests,
       [
         TestItem(
-          id: "MyTests",
+          id: "dummy.MyTests",
```

Comment:
When there is no build system available everything is [lumped in to a dummy target](https://github.com/swiftlang/sourcekit-lsp/blob/af43bae724366d6fb0b5c7f65721d23b97099558/Sources/SKCore/CompilationDatabaseBuildSystem.swift#L127). Is this likely to happen in the real world or is it only a test consideration?

Reply:
This could also happen in practice if you have a non-SwiftPM project with test cases (eg. compilation database). Granted, at that point we don’t have any editor that would know how to run the tests but still, we shouldn’t include `dummy` in the test ID.

Reply:
Fixed